src/guides/v2.2/config-guide/cli/config-cli-subcommands-i18n.md

```diff
@@ -323,11 +323,9 @@ Similar to the preceding example, generate a `.csv` file, but instead of specify
 
 ## Additional information
 
--  [Translations overview]
--  [Translate theme strings]
+-  [Translations]
```

Comment:
Back out the changes in this file.  There's no reason to delete the link to "Translate theme strings", and the title of the `xlate.md` is "Translations overview"